packages/site-content/package.json

```diff
@@ -15,7 +15,8 @@
       "command": "eleventy --config=eleventy.config.prod.cjs",
       "files": [
         "eleventy.config.prod.cjs",
-        "site"
+        "site",
+        "../site-server/src/templates/base.ts"
```

Comment:
I think this should be the `.js` file, not the `.ts` file, otherwise there is a race condition.